site/src/docs/contributing/backend/index.md

```diff
@@ -30,21 +30,19 @@ Run tests in your IDE, and re-run `make rundev` each time you want to see how yo
 
 You have to [install](https://golang.org/doc/install) the latest stable `go` toolchain to run the backend locally.
 
-In order to have working Remark42 installation you need once to copy frontend static files to `./backend/web` directory from `master` docker image, and also copy files from `./templates` to the `./backend` as they are expected to be where application starts:
+In order to have working Remark42 installation you need once to copy frontend static files to `./backend/web` directory from `master` docker image, as it is expected to be where application compiles:
 
 ```shell
 # frontend files
 docker pull umputun/remark42:master
 docker create -ti --name remark42files umputun/remark42:master sh
-docker cp remark42files:/srv/web/ ./backend/
+docker cp remark42files:/srv/web/ ./backend/app/cmd/
```

Comment:
This might be not working anymore as the /web/ directory is now embedded into binary and not present in the Dockerfile. I'll figure out what to do with that a bit later.